winterfell/webapp/src/templates/claimSelectForms.pug

```diff
@@ -9,15 +9,15 @@
         uib-progressbar.progressbar(value="numRequiredCompleted", max="numRequiredForms", type="{{numRequiredCompleted == numRequiredForms ? 'success' : 'info'}}")
           span {{numRequiredCompleted}}/{{numRequiredForms}}
 
-      div.form-wrapper(ng-repeat="(formId, form) in forms")
+      div.form-wrapper(ng-repeat="(formId, form) in allForms", ng-init="myForm = myForms[formId]")
```

Comment:
it allows you to set a variable.
Instead of having `myForms[formId]` everywhere, now I can set it to `myForm` and use that variable instead.